LINAC.h

```diff
@@ -0,0 +1,21 @@
+/* 
```

Comment:
We usually delete the mplab-generated header. We have git blame to keep track of authorship which is more granular anyway.